dnn.go

```diff
@@ -109,3 +109,18 @@ func BlobFromImage(img Mat, scaleFactor float64, size image.Point, mean Scalar,
 
 	return Mat{p: C.Net_BlobFromImage(img.p, C.double(scaleFactor), sz, sMean, C.bool(swapRB), C.bool(crop))}
 }
+
+// GetBlobChannel extracts a single (2d)channel from a 4 dimensional blob structure
+// (this might e.g. contain the results of a SSD or YOLO detection,
+//  a bones structure from pose detection, or a color plane from Colorization)
+//
+func GetBlobChannel(blob Mat, imgidx int, chnidx int) Mat {
+	return Mat{p: C.Net_GetBlobChannel(blob.p, C.int(imgidx), C.int(chnidx))}
+}
+
+// GetBlobSize retrieves the 4 dimensional size information in (N,C,H,W) order
+//
```

Comment:
This was not really needed, and we usually put a period at the end of the line because we write sentences :)

Reply:
ok, your point.

you mean, the function was noot needed ? which functionality does it duplicate ?

Reply:
ah sorry for the brevity of my comment, I meant the extra commented line was not needed.